src/fare-contracts/NoActiveTickets.tsx

```diff
@@ -0,0 +1,24 @@
+import {useTheme} from '@atb/theme';
+import {TicketingTexts, useTranslation} from '@atb/translations';
+import {TicketTilted} from '@atb/assets/svg/color/images';
+import {EmptyState} from '@atb/components/empty-state';
+
+export const NoActiveTickets = () => {
+  const {theme} = useTheme();
+  const {t} = useTranslation();
+
+  return (
+    <EmptyState
+      title={t(
+        TicketingTexts.activeFareProductsAndReservationsTab.noActiveItemsTitle,
+      )}
+      details={t(
+        TicketingTexts.activeFareProductsAndReservationsTab
+          .noActiveItemsDetails,
+      )}
+      illustrationComponent={
+        <TicketTilted height={theme.icon.size.large * 4} />
```

Comment:
I don't think it makes sense to couple this size to icon sizes: If we change the large icon size, I don't think we want this illustration to change size. But maybe we should add sizes for illustrations in the design system at some point.

For now it might be better to hard code a size here?

Reply:
Good point, I just used the biggest icon size I could find, but it wasn't enough 😛
Will fix :+1: